### ISSUE TITLE
Feature: Import Directly Via Foundry

### ISSUE THREAD

**Issue by Golurkcanfly** [ChadKeating/rolladvantage-token-stamp-2-foundry#11]
This feature would mainly be useful for Foundry users hosting on an external server, such as The Forge, allowing users to choose image files already saved in Foundry instead of uploading from a local device.

**Comment by simtudor** [comment]
yes, i agree with this. currently, there doesn't seem to be a way for a player to use this without uploading from their own files. it would be nice if the Actor portrait was automatically put into TA when you open the creator.
